mta.yaml

```diff
@@ -9,7 +9,7 @@ modules:
       memory: 1024M
       disk-quota: 256M
     build-parameters:
-      builder: maven
-      maven-opts:
-        command: [ clean, install ]
+      builder: custom
```

Comment:
was it not working the same before?

Reply:
Yes, Cloud mta build tool does not support` maven-opts` parameter anymore.